services/src/test/resources/spring-test/cdr-client-container.xml

```diff
@@ -137,4 +137,47 @@
     <bean id="sparqlUpdateService" class="edu.unc.lib.dl.sparql.FedoraSparqlUpdateService">
         <property name="fcrepoClient" ref="fcrepoClient" />
     </bean>
+    
+    <bean id="redisServer" class="redis.embedded.RedisServer" init-method="start" destroy-method="stop">
```

Comment:
Very few of our tests use redis, so I would prefer if all of the redis setup were moved either to a separate context file (like we do with fedora) and only imported when needed. Otherwise I believe redis will get started for all of our services integration tests.

Although if you decide to mock redis for your test instead, then this can probably be removed

Reply:
removed it, since i'm mocking